src/settings_type.h

```diff
@@ -68,6 +68,13 @@ enum UseRelayService {
 	URS_ALLOW,
 };
 
+/** Right-click to close window actions. */
+enum RightClickClose {
```

Comment:
I think it still needs `: uint8` hereto force the enum type as the settings type is SLE_UINT8.